docs/user-guide/mvd-configuration.md

```diff
@@ -175,36 +189,42 @@ For more information, see [Logging Utility](../extend/extend-desktop/mvd-logutil
+    port: 8542
+    crossMemoryServerName: ZWESIS_STD
+    # The following is not secure & should only be used with AT-TLS and tls: false
+    agent:
+      http:
+        ipAddresses: ["127.0.0.1"]
+        attls: true
 ```
 
 ### Connecting App Server to ZSS
 
-When running the App Server, simply specify a few flags to declare which ZSS instance the App Server will proxy ZSS requests to:
+When running the App Server script directly, simply specify a few flags to declare which ZSS instance the App Server will proxy ZSS requests to:
 
 - *-h*: Declares the host where ZSS can be found. Use as "-h \<hostname\>"
 - *-P*: Declares the port at which ZSS is listening. Use as "-P \<port\>"
 
 ### Configuring ZSS for HTTPS
-To secure ZSS communication, you can use Application Transparent Transport Layer Security (AT-TLS) to enable Hyper Text Transfer Protocol Secure (HTTPS) communication with ZSS.
 
-Before you begin, you must have a basic knowledge of your security product, e.g. RACF, and AT-TLS, and you must have Policy Agent configured. For more information on [AT-TLS](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.1.0/com.ibm.zos.v2r1.halx001/transtls.htm) and [Policy Agent](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.2.0/com.ibm.zos.v2r2.halz002/pbn_pol_agnt.htm), see the [z/OS Knowledge Center](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.2.0/com.ibm.zos.v2r2/en/homepage.html).
+By default, ZSS is enabled in HTTPS and doesn't require an advanced configuration.
+
+If you want to use RACF or AT-TLS (which requires ZSS to be in HTTP mode), you must have a basic knowledge of your security product and you must have Policy Agent configured. For more information on [AT-TLS](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.1.0/com.ibm.zos.v2r1.halx001/transtls.htm) and [Policy Agent](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.2.0/com.ibm.zos.v2r2.halz002/pbn_pol_agnt.htm), see the [z/OS Knowledge Center](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.2.0/com.ibm.zos.v2r2/en/homepage.html).
 
 You must have the authority to alter security definitions related to certificate management, and you must be authorized to work with and update the Policy Agent.
 
-To configure HTTPS communication between ZSS and the Zowe App Server, you need a key ring which contains the ZSS server certificate and its Certificate Authority (CA) certificate. You can use an internal CA to create the ZSS server certificate, or you can buy the ZSS server certificate from a well-known commercial Certificate Authority. Next you define an AT-TLS rule which points to the key ring used by the ZSS server. Then you copy the CA certificate to the Zowe App Server key store and update the Zowe App Server configuration file.
+By default, HTTPS communication between ZSS and App Server is done via a server certificate, its Certificate Authority (CA) certificate, and keystore generated during the Zowe initialization step (`zwe init`). 
```

Comment:
Remove 225

227 should say "If you are going to use AT-TLS, you will need to set up TLS rule and TLS keyring. The next section will cover that information."